pkgs/applications/networking/instant-messengers/webcord/webcord-vencord/default.nix

```diff
@@ -11,6 +12,10 @@ webcord.overrideAttrs (old: {
     })
   ];
 
+  # Webcord has updated to electron 25, but that causes a segfault
+  # when launching webcord-vencord on wayland, so downgrade it for now.
+  electron_25 = electron_24;
```

Comment:
This doesn't actually do anything since `overrideAttrs` is for overriding attributes passed to mkDerivation.

To override electron_25 we need to use `override`

I don't know if there's a better way to do this, but that works:

```nix
{ webcord
, substituteAll
, lib
, vencord-web-extension
, electron_24
,
}:
(webcord.overrideAttrs (old: {
  patches = (old.patches or [ ]) ++ [
    (substituteAll {
      src = ./add-extension.patch;
      vencord = vencord-web-extension;
    })
  ];

  meta = with lib;
    old.meta
    // {
      description = "Webcord with Vencord web extension";
      maintainers = with maintainers; [ FlafyDev NotAShelf ];
    };
})).override {
  # Webcord has updated to electron 25, but that causes a segfault
  # when launching webcord-vencord on wayland, so downgrade it for now.
  electron_25 = electron_24;
}
```

Reply:
wups I always make this mistake will fix